### PR TITLE
Update login.form.tpl

### PR DESCRIPTION
**Pull request by rmarzocchi84**
Login form responsive

### CODE REVIEW

File: lib/jelix-modules/jauth/templates/login.form.tpl [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {if ! $isLogged}
 
 <form action="{formurl 'jauth~login:in'}" method="post" id="loginForm">
-      <fieldset>
+      <f<div class="container">
```
Comment (rldhont) [inline lib/jelix-modules/jauth/templates/login.form.tpl]: Can you remove `<f`

**Comment by rldhont** [comment]
@rmarzocchi84 the file to modify is not the one you updated.
The right file is `lizmap/var/themes/default/jcommunity/login.tpl`
Can you do the update ?

**Comment by rldhont** [comment]
I have tested the changes to `lizmap/var/themes/default/jcommunity/login.tpl` and it is not the right way.

**Comment by rldhont** [comment]
@rmarzocchi84 I have found how to fix responsive login page.

**Comment by rmarzocchi84** [comment]
Sorry for my not correct suggestion. Unfortunately I never used this version of bootstrap (I have experience only with v.3 and v.4)

**Comment by rldhont** [comment]
@rmarzocchi84 thanks to point the issue

**Pull request closed without merging**